xdem/terrain.py

```diff
@@ -414,12 +454,28 @@ def get_terrain_attribute(
 
     output_attributes = [terrain_attributes[key].reshape(dem.shape) for key in attribute]
 
+    if isinstance(dem, gu.Raster):
```

Comment:
we're going to write this line quite often I think! but it's not too much, it's fine :smile: !